Rajitha/google/google.css

```diff
@@ -0,0 +1,92 @@
+.topnav {
+  background-color: white;
+}
+.topnav a {
```

Comment:
avoid overriding html tags 

add a class name to <a> tag

Reply:
Modified and added class name